### DISCUSSION TITLE
Feedback

### DISCUSSION THREAD

**Discussion by bameyrick** [ONSdigital/design-system#1388] [Feedback]
Use this place to discuss the [feedback pattern](https://ons-design-system.netlify.com/patterns/feedback) in the [ONS Design System](https://ons-design-system.netlify.com)

**Comment by jrbarnes9** [comment]
Feedback Mailto link contains a full stop so invalid when copied and pasted.

**Comment by rmccar** [comment]
https://github.com/ONSdigital/design-system/blob/3fbea5e0371e96926ef7e1fa06f02a893773bde7/src/tests/spec/feedback/feedback.spec.js#L89 Test needs looking at as always passes. I needs to test that the message is visible not if it exists

**Comment by jrbarnes9** [comment]
Can't add classes to feedback div to adjust margins, and border doesn't match the default header theme colour

**Comment by Al-Davies** [comment]
**tested as part of eQ business survey usability testing Oct 21**

1. users would expect some sort of rating scale, or specific questions within feedback
2. a free text only box - users felt this wouldn't give us useful feedback, that we would need to 'tease' useful feedback from them (by asking specific questions)
